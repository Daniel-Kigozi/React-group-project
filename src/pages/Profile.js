import React from 'react';

const Profile = () => (
  <div className="myProfile-container" />
);

export default Profile;