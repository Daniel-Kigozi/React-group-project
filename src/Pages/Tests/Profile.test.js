import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Profile from '../Profile';
import store from '../../Redux/configureStore';

it('renders correctly', () => {
  const profile = renderer
    .create(
      <Provider store={store}><Profile /></Provider>,
    )
    .toJSON();
  expect(profile).toMatchSnapshot();
});
