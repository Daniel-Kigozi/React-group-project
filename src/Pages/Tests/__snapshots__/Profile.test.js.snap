// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rocketListDiv"
>
  <span
    className="profile-view"
  >
    <div
      className="missions-profile"
    >
      <h1>
        My Missions
      </h1>
      <div
        className="our-missions"
      >
        <p>
          You have no reserved missions
        </p>
      </div>
    </div>
  </span>
  <span
    className="profile-view"
  >
    <h1
      className="myRocketsHeader"
    >
      My Rockets
    </h1>
    <ul
      className="rocketListContainer"
    />
  </span>
</div>
`;
